recipes/hjson/meta.yaml

```diff
@@ -0,0 +1,56 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+{% set name = "hjson" %}
+{% set version = "3.0.1" %}
+{% set sha256 = "1d1727faa6aaef2973921877125a3ab7c5f6d34b93233179d01770f41fab51f9" %}
+# sha256 is the prefered checksum -- you can get it for a file with:
+#  `openssl sha256 <file name>`.
+# You may need the openssl package, available on conda-forge
+#  `conda install openssl -c conda-forge``
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python
+    - pip
+  run:
+    - python
+
+test:
```

Comment:
You might consider testing the command line tools as well. I have a memory of this not being handled well when using `noarch: python`, but that might have just been tools built as entry points:

https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#test-commands